pandas/core/series.py

```diff
@@ -547,6 +547,71 @@ def __len__(self):
         return len(self._data)
 
     def view(self, dtype=None):
+        """
+        Create a new view of the Series.
+
+        This function will return a new Series with a view of the same
+        underlying values in memory, optionally reinterpreted with a new data
+        type. The new data type must preserve the same size in bytes as to not
+        cause index misalignment.
+
+        Parameters
+        ----------
+        dtype : data type
+            Data type object or one of their string representations.
+
+        Returns
+        -------
+        Series
+            A new Series object as a view of the same data in memory.
+
+        See Also
+        --------
+        numpy.ndarray.view : Equivalent numpy function to create a new view of
+            the same data in memory.
+
+        Notes
+        -----
+        Series are instantiated with ``dtype=float64`` by default. While
+        ``numpy.ndarray.view()`` will return a view with the same data type as
+        the original array, ``Series.view()`` (without specified dtype)
+        will try using ``float64`` and may fail if the original data type size
+        in bytes is not the same.
```

Comment:
I don't really like this note (I mean, the explanation is fine and correct, but we should just correct the code to not do this), but fine for now and I will open an issue to eg disallow not specifying a dtype, or if dtype=None to use `self.dtype`

Reply:
see #20251 yes let's remove most of this text. This method should be deprecated anyhow.

Reply:
The current text is correct, so let's remove *when* we deprecate it.

Reply:
I don't like this odd behavior either, but I don't think it's necessary to deprecate the function altogether. Wouldn't simply passing the current `dtype` (if the argument is `None`) to the constructor solve the oddity?